FILES/N_0x10144267dd22866c.md

```diff
@@ -1,29 +1,22 @@
 ---
 ns: FILES
+aliases: ["_0x10144267DD22866C"]
 ---
-## _0x10144267DD22866C
+## GET_TATTOO_SHOP_DLC_ITEM_INDEX
 
 ```c
 // 0x10144267DD22866C
-int _0x10144267DD22866C(Hash overlayHash, Any p1, int character);
-```
-
-Returns some sort of index/offset for overlays/decorations.
-
-```
-Character types:
-0 = Michael,
-1 = Franklin,
-2 = Trevor,
-3 = MPMale,
-4 = MPFemale
+int GET_TATTOO_SHOP_DLC_ITEM_INDEX(cs_type(Hash) int character, cs_type(Any) int collection, cs_type(int) Hash preset);
```

Comment:
Hey, these are all `int`. For `Any` to `int` you don't need `cs_type` by the way.

So ideally it would look like this:
```c
int GET_TATTOO_SHOP_DLC_ITEM_INDEX(cs_type(Hash) int character, int collection, int preset);
```

Reply:
Thanks! I changed it